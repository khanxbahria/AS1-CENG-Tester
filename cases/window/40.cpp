#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Window* wind = new Window();
Tab J = Tab("lll", "xx","ee");
wind->closeTab();
Tab G = Tab("r", "vv","xxx");
wind->changeActiveTabTo(3);
Tab K = Tab("lll", "i","xx");
wind->changeActiveTabTo(1);
Tab D = Tab("ggg", "zzz","ii");
wind->changeActiveTabTo(3);
Tab X = Tab("ddd", "c","a");
wind->closeTab();
Tab R = Tab("h", "bbb","ddd");
wind->newTab(D);
Tab Q = Tab("v", "vvv","nnn");
wind->changeActiveTabTo(2);
Tab Y = Tab("nn", "www","pp");
wind->newTab(G);
Tab I = Tab("aa", "pp","fff");
wind->closeTab();
Tab B = Tab("nnn", "xx","d");
wind->newTab(B);
Tab C = Tab("nnn", "x","ee");
wind->print();
Tab S = Tab("qqq", "z","ooo");
wind->getActiveTab();
Tab V = Tab("sss", "bbb","bb");
wind->closeTab();
wind->getActiveTab();
wind->print();
wind->print();
Tab O = Tab("z", "kk","hh");
wind->getActiveTab();
Tab P = Tab("nnn", "w","v");
wind->moveActiveTabTo(4);
Tab F = Tab("kkk", "j","d");
wind->getActiveTab();
Tab T = Tab("ddd", "yy","jj");
wind->isEmpty();
Tab M = Tab("jj", "cc","ll");
wind->moveActiveTabTo(2);
wind->changeActiveTabTo(2);
wind->changeActiveTabTo(1);
wind->closeTab();
wind->print();
wind->changeActiveTabTo(2);
wind->closeTab();
wind->print();
Tab H = Tab("jjj", "j","nnn");
wind->closeTab();
wind->moveActiveTabTo(4);
wind->getActiveTab();
wind->print();
wind->moveActiveTabTo(4);
Tab Z = Tab("e", "c","www");
wind->moveActiveTabTo(2);
wind->changeActiveTabTo(4);
wind->closeTab();
wind->closeTab();
wind->print();
wind->changeActiveTabTo(2);
wind->print();
Tab L = Tab("ppp", "u","f");
wind->closeTab();
Tab E = Tab("vvv", "v","gg");
wind->newTab(C);
wind->closeTab();
Tab W = Tab("gg", "kkk","rr");
wind->getActiveTab();
wind->changeActiveTabTo(5);
wind->newTab(E);
wind->isEmpty();
wind->print();
wind->print();
wind->isEmpty();
wind->changeActiveTabTo(4);
wind->moveActiveTabTo(4);
wind->getActiveTab();
wind->changeActiveTabTo(4);
wind->closeTab();
wind->moveActiveTabTo(0);
wind->newTab(F);
wind->moveActiveTabTo(4);
wind->changeActiveTabTo(0);
wind->isEmpty();
wind->changeActiveTabTo(0);
wind->print();
wind->print();
wind->changeActiveTabTo(4);
wind->closeTab();
wind->getActiveTab();
wind->changeActiveTabTo(3);
wind->newTab(H);
wind->print();
wind->newTab(I);
Tab N = Tab("jj", "sss","p");
wind->isEmpty();
Tab U = Tab("uu", "z","uu");
wind->closeTab();
wind->newTab(J);
wind->isEmpty();
wind->moveActiveTabTo(1);
wind->newTab(K);
wind->moveActiveTabTo(5);
wind->getActiveTab();
wind->print();
wind->newTab(L);
wind->isEmpty();
wind->print();
wind->getActiveTab();
wind->changeActiveTabTo(3);
wind->getActiveTab();
wind->print();
wind->print();
wind->print();
wind->changeActiveTabTo(4);
wind->print();
wind->print();
wind->closeTab();
wind->print();
wind->closeTab();
wind->isEmpty();
wind->getActiveTab();
Tab A = Tab("www", "f","ii");
wind->getActiveTab();
wind->closeTab();
wind->newTab(A);
delete wind;
}
	return 0;
}