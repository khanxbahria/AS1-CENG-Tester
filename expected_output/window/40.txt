The active tab is tab 1
zzz
ggg
ii

xx
nnn
d

The active tab is tab 0
zzz
ggg
ii

The active tab is tab 0
zzz
ggg
ii

Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0
v
vvv
gg

The active tab is tab 0
v
vvv
gg

The active tab is tab 0
j
kkk
d

The active tab is tab 0
j
kkk
d

The active tab is tab 0
j
jjj
nnn

The active tab is tab 2
j
jjj
nnn

xx
lll
ee

i
lll
xx

The active tab is tab 3
j
jjj
nnn

xx
lll
ee

i
lll
xx

u
ppp
f

The active tab is tab 3
j
jjj
nnn

xx
lll
ee

i
lll
xx

u
ppp
f

The active tab is tab 3
j
jjj
nnn

xx
lll
ee

i
lll
xx

u
ppp
f

The active tab is tab 3
j
jjj
nnn

xx
lll
ee

i
lll
xx

u
ppp
f

The active tab is tab 3
j
jjj
nnn

xx
lll
ee

i
lll
xx

u
ppp
f

The active tab is tab 3
j
jjj
nnn

xx
lll
ee

i
lll
xx

u
ppp
f

The active tab is tab 2
j
jjj
nnn

xx
lll
ee

i
lll
xx

